File: be/src/main/java/sg/flow/models/auth/AccessTokenRefreshRequest.java
package sg.flow.models.auth;

import lombok.Data;

@Data
public class AccessTokenRefreshRequest {
    
    private String refreshToken;

    public String getRefreshToken() {
        return refreshToken;
    }

    public void setRefreshToken(String refreshToken) {
        this.refreshToken = refreshToken;
    }
}

File: be/src/main/kotlin/sg/flow/models/auth/AccessTokenRefreshRequest.kt
package sg.flow.models.auth

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Size

data class AccessTokenRefreshRequest(
    @field:NotBlank(message="Refresh Token cannot be blank")
    @field:Size(min = 64, max = 64, message = "Refresh Token should be of size 64")
    var refreshToken: String
)
